<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>players</class>
 <widget class="QDialog" name="players">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="21" column="0" colspan="3">
    <widget class="QSlider" name="time_slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="bt_field2">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="bt_field3">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QPushButton" name="bt_field4">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QPushButton" name="bt_field5">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QSpinBox" name="in_minutes">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QSpinBox" name="in_seconds">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QPushButton" name="bt_close">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="bt_field1">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="lb_spiega">
     <property name="font">
      <font>
       <family>Lucida Console</family>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="bt_res_10">
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="bt_res_09">
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="bt_res_08">
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="bt_res_07">
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="bt_change">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>&lt;-&gt;</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QPushButton" name="bt_res_06">
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QPushButton" name="bt_res_05">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QPushButton" name="bt_res_04">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QPushButton" name="bt_res_03">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="bt_res_02">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="bt_res_01">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(85, 255, 0);
selection-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QPushButton" name="bt_my_timeout">
     <property name="text">
      <string>TO my</string>
     </property>
    </widget>
   </item>
   <item row="23" column="2">
    <widget class="QPushButton" name="bt_oth_tmout">
     <property name="text">
      <string>TO oth</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QPushButton" name="bt_f_none">
     <property name="text">
      <string>Nobody</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QPushButton" name="bt_f_coach">
     <property name="text">
      <string>Coach</string>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QPushButton" name="bt_f_bench">
     <property name="text">
      <string>Bench</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
